en/flight_controller/raspberry_pi_navio2.md

```diff
@@ -52,121 +59,6 @@ Edit `/boot/config.txt` by commenting the line enabling the `navio-rgb` overlay.
 #dtoverlay=navio-rgb
 ```
 
-## Changing Hostnames
```

Comment:
This is not needed/advisable any more? I.e why do we no longer want to avoid conflicts with other RPis on the network?

Reply:
The pre-configured image now comes with a custom hostname and no longer uses the standard “raspberrypi” hostname which would create conflict.

Reply:
Thank you for clarification.